src/views/virtualmachinesinstance/details/tabs/details/components/Services/Services.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+import { ServiceModel } from '@kubevirt-ui/kubevirt-api/console';
+import { IoK8sApiCoreV1Service } from '@kubevirt-ui/kubevirt-api/kubernetes/models';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { getServicesForVmi } from '@kubevirt-utils/resources/vmi';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import { ServicesList } from '@openshift-console/dynamic-plugin-sdk-internal';
+import { Title } from '@patternfly/react-core';
+import { LinkIcon } from '@patternfly/react-icons';
+
+const Services = ({ vmi, pathname }) => {
+  const { t } = useKubevirtTranslation();
+
+  const [services, loaded] = useK8sWatchResource<IoK8sApiCoreV1Service[]>({
+    kind: ServiceModel?.kind,
```

Comment:
kind is about to be depracated, use groupVersionKind: modelToGroupVersionKind(ServiceModel) instead

Reply:
great, changed.